### PR TITLE
ref(ui): alert badge

### PR DESCRIPTION
**Pull request by robinrendle**
- Removed duplicate AlertBadge on the Project Details page
- Moved AlertBadge component to `/components` so that it’s more obvious it’s a reusable component
- Used IconDiamond for the AlertBadge instead of CSS

## Before

CleanShot 2022-02-16 at 15 13 56
CleanShot 2022-02-16 at 15 13 50

## After
CleanShot 2022-02-17 at 11 12 19
CleanShot 2022-02-17 at 11 12 29

### CODE REVIEW

**Comment by doralchan** [comment]
Wait wat? Why aren't you using IconDiamond

**Review by doralchan — DISMISSED** [review]
I'm confused about why you're not using IconDiamond instead of creating iconAlertCritical, iconAlertIssues, iconAlertResolved and iconAlertWarning

**Comment by robinrendle** [comment]
@doralchan Yeah, you’re right. I’ve removed those icons and switched the background of AlertBadge to use IconDiamond.

File: static/app/components/alertBadge.tsx [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+  IconExclamation,
+  IconFire,
+  IconIssues,
+} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
 import {Color} from 'sentry/utils/theme';
 
-import {IncidentStatus} from './types';
+import {IncidentStatus} from '../views/alerts/types';
```
Comment (scttcper) [inline static/app/components/alertBadge.tsx:14]:
```suggestion
import {IncidentStatus} from 'sentry/views/alerts/types';
```

**Review by scttcper — APPROVED** [review]

**Review by scttcper — APPROVED** [review]

**Pull request merged**